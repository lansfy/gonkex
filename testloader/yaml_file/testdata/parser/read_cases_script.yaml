- name: loader MUST substitute beforeScriptArgs/afterRequestScript to script path
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  beforeScript:
    path: '/some/path/{{ .foo }}'
  afterRequestScript:
    path: '/some/path/{{ .bar }}'
  cases:
    - beforeScriptArgs:
        foo: "script1.sh"
      afterRequestScriptArgs:
        bar: "otherscript1.sh"
    - beforeScriptArgs:
        foo: "script2.sh"
      afterRequestScriptArgs:
        bar: "otherscript2.sh"
