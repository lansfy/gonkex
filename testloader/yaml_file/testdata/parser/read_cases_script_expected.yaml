- GetName: 'loader MUST substitute beforeScriptArgs/afterRequestScript to script path #1'
  GetMethod: GET
  Path: /some/path
  GetResponses:
    200: "aaabbb"
  BeforeScript:
    CmdLine: /some/path/script1.sh
  AfterRequestScript:
    CmdLine: /some/path/otherscript1.sh
  GetFileName: testdata/parser/read_cases_script.yaml
  OneOfCase: true
  FirstTestInFile: true

- GetName: 'loader MUST substitute beforeScriptArgs/afterRequestScript to script path #2'
  GetMethod: GET
  Path: /some/path
  GetResponses:
    200: "aaabbb"
  BeforeScript:
    CmdLine: /some/path/script2.sh
  AfterRequestScript:
    CmdLine: /some/path/otherscript2.sh
  GetFileName: testdata/parser/read_cases_script.yaml
  OneOfCase: true
  LastTestInFile: true
